build-tests/api-documenter-test/etc/api-documenter-test.api.json

```diff
@@ -17,7 +17,7 @@
         {
           "kind": "Class",
           "canonicalReference": "(DocBaseClass:class)",
-          "docComment": "/**\n * Example base class\n *\n * @public\n */\n",
+          "docComment": "/**\n * Example base class\n *\n * {@docCategory DocBaseClass}\n *\n * @public\n */\n",
```

Comment:
@iclanton I bet this change to the documentation is causing the addition of a blank line. `MarkdownDocumenter` does not know how to handle inline custom tags and it skips `{@docCategory DocBaseClass}` and is left with `\n` right after. So this is specific to only this case and this test project. I don't consider it as a regression. Perhaps a `MarkdownDocumenter` bug that needs to be fixed in a separate PR.

Reply:
Cool.